### PR TITLE
Typo errors fixed, added VIDWallet into the implementers table

### PR DESCRIPTION
**Pull request by andreasabr**
Fixed a few typos and also added VIDwallet to the implementers table

### CODE REVIEW

**Review by georgepadayatti — APPROVED** [review]

**Comment by georgepadayatti** [comment]
LGTM once the merge conflicts are fixed.

**Review by lalc — APPROVED** [review]
This looks ok

**Comment by lalc** [comment]
@andreasabr I have added you as an author as discussed

**Pull request merged**
